routes/web.php

```diff
@@ -17,6 +17,20 @@
     return view('welcome');
 });
 
+Route::get('celebrities', [CelebrityController::class, 'index'])->name('celebrities.index');
+
+Route::get('celebrity/create', [CelebrityController::class, 'create'])->name('celebrities.create');
+
+Route::get('celebrities/{celebrity}/edit', [CelebrityController::class, 'edit'])->name('celebrities.edit');
+
+Route::get('celebrities/{celebrity}', [CelebrityController::class, 'show'])->name('celebrities.show');
+
+Route::post('celebrities', [CelebrityController::class, 'store'])->name('celebrities.store');
+
+Route::put('celebrities/{celebrity}', [CelebrityController::class, 'update'])->name('celebrities.update');
+
+Route::delete('celebrities/{celebrity}', [CelebrityController::class, 'destroy'])->name('celebrities.destroy');
+
```

Comment:
I think that chat a bit yesterday about this ;D But there are some of these routes that should be only seen by authenticated users. 
You can use the `Route::group` and use the middleware `auth` to make some of these routes only accessible to authenticated users.